src/utils/mapItemsWithComponents.ts

```diff
@@ -40,16 +41,22 @@ export function mapItemsWithComponents<T>(
         ...rest,
         component: root,
         children: webComponentChildren,
+        __item: item,
```

Comment:
This line doesn't seem to be covered by tests added in the PR, as they only cover `else` block.

Reply:
Updated the test case